Xplat/src/main/java/vazkii/botania/mixin/BeeMixin.java

```diff
@@ -35,10 +35,10 @@ protected BeeMixin(EntityType<? extends Animal> type, Level worldIn) {
 	 */
 	@Inject(
 		method = "isFlowerValid", cancellable = true,
-		at = @At(value = "FIELD", target = "Lnet/minecraft/world/entity/animal/Bee;level:Lnet/minecraft/world/level/Level;", ordinal = 1)
+		at = @At(value = "HEAD")
```

Comment:
any reason for this?

Reply:
isFlowerValid no longer accesses the level field directly, instead using the method. i couldn't get it to work with targeting invoke, so i changed it to head; i was of the impression this would produce the same behaviour.

Reply:
i got the invoke mixin to work, now it's closer to how it was before